README.md

```diff
@@ -61,7 +61,7 @@ If for any reason you have questions or need to contact us, you can find us on t
 Once you have the above list of things, it's typically a good idea to build the website from your local computer to make sure it works before you make any changes. To do this, complete the following steps:
 
 1. Navigate to your local `monero-site` repository.
-2. Serve the website: `jekyll serve`
+2. Serve the website: `bundle exec jekyll serve`
```

Comment:
It is still possible to avoid "bundle exec" if you use the proper version of ruby (2.4.1) instead of the latest 2.5.

Reply:
I couldn't run it on a newly setup VM without the `bundle exec`.